### PR TITLE
Added May Long Challenge 2021 to 3]. Competitive Programming/CodeChef

### PR DESCRIPTION
**Pull request by aroravansh**
Solutions of May long challenge 2021 in CPP

### CODE REVIEW

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/ISS.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/ISS.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/MODEQ.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/*
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/MODEQ.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/TCTCTOE.cpp [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+XOX
+OX_
+XO_
+Sample Output
+2
+3
+1
+*/
+
+#include<bits/stdc++/h>
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/TCTCTOE.cpp:31]: Error
change `#include<bits/stdc++/h>` to `#include<bits/stdc++.h>`

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/THOUSES.cpp [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    }
+
+    ch.clear();
+    for (auto x : adj[now]) {
+        if (x != lst) ch.eb(subval[x], x);
+    }
+    sort(RALL(ch));
+
+    int tok = 1;
+    for (auto [_val, id] : ch) val[id] = tok++;
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/THOUSES.cpp:95]: Error
Resolve this question

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/VPATH.cpp [hunk 5]
```diff
@@ -0,0 +1,118 @@
+/*
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/VPATH.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/XOREQUAL.cpp [hunk 6]
```diff
@@ -0,0 +1,44 @@
+/*
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/XOREQUAL.cpp:1]: Accepted

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Change two solutions `TCTCTOE.cpp` (change 1st line of code) & `THOUSES.cpp`

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/THOUSES.cpp (hunk 4 above)
Comment (aroravansh) [reply 3]. Competitive Programming/CodeChef/May Long Challenge 2021/THOUSES.cpp:95]: This code works C++17 gcc 9.1 not on C++14

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/TCTCTOE.cpp [hunk 7]
```diff
@@ -0,0 +1,78 @@
+/*
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/TCTCTOE.cpp:1]: Accepted

File: 3]. Competitive Programming/CodeChef/May Long Challenge 2021/THOUSES.cpp [hunk 8]
```diff
@@ -0,0 +1,127 @@
+/*
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/CodeChef/May Long Challenge 2021/THOUSES.cpp:1]: Accepted

**Review by AkashSingh3031 — COMMENTED** [review]
All Accepted

**Pull request merged**